_tuning-your-cluster/seperate-index-and-search-workloads.md

```diff
@@ -148,11 +148,13 @@ The `cluster.routing.search_replica.strict` setting supports the following optio
 
 Use the `auto_expand_search_replicas` index setting to automatically scale search replicas based on the number of available search nodes in the cluster. For more information, see [Index settings]({{site.url}}{{site.baseurl}}/install-and-configure/configuring-opensearch/index-settings/#dynamic-index-level-index-settings).
 
-### Turn off write workloads
+### Turn off write workloads with search-only mode
 
-You can use the `_scale` API to turn off primary shards and write replicas if you don't expect any writes to an index. In write-once, read-many scenarios (like log analytics), you can scale down primary and write replicas, leaving only search replicas active to free up resources.
+You can use the `_scale` API to turn off primary shards and write replicas for an index when you don't need to write to it. This approach works well for write-once, read-many scenarios like log analytics, where you can reduce resource usage by keeping only search replicas active.
 
-The following [scale]({{site.url}}{{site.baseurl}}/api-reference/index-apis/scale/) request turns off write replicas:
+Before enabling search-only mode, we recommend setting `cluster.remote_store.state.enabled` to `true`. For more information about recovery scenarios, see [Search replica recovery scenarios](#search-replica-recovery-scenarios).
```

Comment:
This implies that this is a dynamic setting if its 'before enabling search-only mode' but `cluster.remote_store.state.enabled` is a final setting similar to enabling remote store.  Can we add a note on why we suggest this setting for rw split earlier in the documentation?

Reply:
Make sense, let me move this note.

Reply:
Updated in my latest commit, can you please check again ?